src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1422,10 +1420,6 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
         $definition->replaceArgument(0, $config['email_validation_mode']);
 
         if (\array_key_exists('enable_annotations', $config) && $config['enable_annotations']) {
-            if (!$this->annotationsConfigEnabled && \PHP_VERSION_ID < 80000) {
```

Comment:
here too, annotation still has to be activated explicitly, since doctrine annotation is not always installed. or is it not necessary?

Reply:
same here, `PHP_VERSION_ID < 80000` will always be false and the block can be removed